package.json

```diff
@@ -63,7 +63,7 @@
         "start": "yarn start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config conf/webpack.config.js --mode development",
         "start:examples": "yarn setup; EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config conf/styleguide.config.js --mode development",
-        "start:npm": "yarn setup; npm-run-all build:prod:npm build:dev:es",
+        "start:npm": "npm-run-all -s clean build:i18n build:prod:npm copy:styles build:dev:es",
```

Comment:
What specific issue does this address? It looks like `yarn setup` is already doing everything here except `-s` and `copy:styles`.

Reply:
`setup` also running the install step, which seems unnecessary for a dev command.

Reply:
Rebased